### DISCUSSION TITLE
MongoDB & Unraid Help

### DISCUSSION THREAD

**Discussion by rockbo842** [Tzahi12345/YoutubeDL-Material#710] [General]
I've tried setting up MongoDB in Unraid per the instructions found here [here](https://github.com/Tzahi12345/YoutubeDL-Material/wiki/unRAID#mongodb-database-setup) but continue to get a connection failed error in YoutubeDL-Material.

I noticed that the instructions for setting up MongoDB said to "remove the values from the template for the admin user and password".  Neither of those fields appeared in the container editor.  Are those extra parameters that I have to include?

**Comment by the7egend** [comment]
Did you set your network type to 'Host' in the docker container for MongoDB, if so and it won't connect over 127.0.0.1, then you might be better off creating a custom docker network and setting both YouTubeDL-Material and the MongoDB both to it then just connecting to the database in YouTubeDL-Material via: mongodb://MongoDB:27017/?compressors=zlib

Using a custom docker network just lets all your containers talk to each other using their container names, so it makes it super easy.

**Comment by rockbo842** [comment]
@the7egend Thanks for the reply.  That may be the issue.  I have MongoDB set to "Host" but I also have YouTubeDL-Material running in a custom network so that it can borrow the VPN connection of another container.

I tried moving MongoDB to the same custom network to no avail.  Is there a way to have YouTubeDL-Material connect directly through a VPN so I don't have to use the custom network?
